asn1/LedgerLedgerLotocol.asn

```diff
@@ -11,30 +11,40 @@ IMPORTS
     UInt256
     FROM GenericTypes
 
+    SideProtocolDate
+    FROM GenericPacket
+
+    InterledgerPacket
+    FROM GenericPacket
+
     Address,
     Timestamp
     FROM InterledgerTypes
 ;
 
-SideProtocolData ::= SEQUENCE OF SEQUENCE {
-  protocolName IA5String,
-  protocolData OCTET STRING
-}
-
-PacketRequest ::= SEQUENCE {
+FreeRequest ::= SEQUENCE {
+  fromId UInt128,
```

Comment:
I think it's confusing to use ILP addresses in a context where most ILP addresses would be "valid but not local to this connection, so sort of invalid anyway".`.from` and `.to` need to be local to the connection in practice, so we might as well say so in the protocol as well. See also https://github.com/interledger/rfcs/pull/261#discussion_r131582148